packages/devtools_app/lib/src/shared/vm_flags.dart

```diff
@@ -21,40 +21,40 @@ const profiler = 'profiler';
 const profilePeriod = 'profile_period';
 
 class VmFlagManager extends Disposer {
-  VmServiceWrapper get service => _service;
-  VmServiceWrapper _service;
+  VmServiceWrapper? get service => _service;
+  VmServiceWrapper? _service;
 
   ValueListenable get flags => _flags;
-  final _flags = ValueNotifier<FlagList>(null);
+  final _flags = ValueNotifier<FlagList?>(null);
 
-  final _flagNotifiers = <String, ValueNotifier<Flag>>{};
+  final _flagNotifiers = <String?, ValueNotifier<Flag?>>{};
 
-  ValueNotifier<Flag> flag(String name) {
+  ValueNotifier<Flag?>? flag(String name) {
     return _flagNotifiers.containsKey(name) ? _flagNotifiers[name] : null;
   }
 
   Future<void> _initFlags() async {
-    final flagList = await service.getFlagList();
+    final FlagList? flagList = await service!.getFlagList();
```

Comment:
this can be non-nullable. VmService.getFlagList does not return a nullable type

Reply:
done